src/dmd/dmodule.d

```diff
@@ -381,6 +381,7 @@ extern (C++) final class Module : Package
     Dsymbols* decldefs;         // top level declarations for this Module
 
     Modules aimports;           // all imported modules
+    Strings astringImports;     // all string-imported files
```

Comment:
Perhaps this can be an array of `const(char)[]` and an ABI compatible struct on the C++ side.

Reply:
Same comment about the status quo. I'd like to replace the `Array` aliases as much as the next guy, but I don't think we should put that kind of weight on this PR.

Reply:
I also think that this is not really in the scope of this PR, and I think a "standard" way to do it (perhaps with predefined structs for most common array typs) should be defined first and a refactor PR could be done with that in place.

Reply:
Then it will never happen and at the same time we’re adding more code that needs to change later 😞